src/Auth/signUpContainer.tsx

```diff
@@ -0,0 +1,81 @@
+import React, {useState} from 'react';
+import {Button, TextInput} from 'react-materialize';
+import {auth} from '../Firebase/firebase';
+import {useNavigate} from 'react-router-dom';
+import './style/style.css';
+
+export const SignUpContainer = (): JSX.Element => {
+    const [confirmPassword, setConfirmPassword] = useState('')
+    const [password, setPassword] = useState('')
+    const [email, setEmail] = useState('')
+    const navigate = useNavigate();
+
+    function createEmailHandler(email: string) {
+        setEmail(email)
+    }
+
+    function createPasswordHandler(password: string) {
+        setPassword(password)
+    }
+    function confirmPasswordHandler(password: string) {
+        setConfirmPassword(password)
+    }
+
+    function backToAuthHandler() {
```

Comment:
You have different naming convention for different component, try to use the same naming inside of one project